examples/CheckFirmwareVersion/CheckFirmwareVersion.ino

```diff
@@ -0,0 +1,59 @@
+/*
+ * This example check if the firmware loaded on the NINA module
+ * is updated.
+ *
+ * Circuit:
+ * - Board with NINA module (Arduino MKR WiFi 1010, MKR VIDOR 4000 and UNO WiFi Rev.2)
+ *
+ * Created 17 October 2018 by Riccardo Rosario Rizzo
+ * This code is in the public domain.
+ */
+#include <SPI.h>
+#include <WiFiNINA.h>
+
+void setup() {
+  // Initialize serial
+  Serial.begin(9600);
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB port only
+  }
+
+  // Print a welcome message
+  Serial.println("WiFiNINA firmware check.");
+  Serial.println();
+
+  // check for the WiFi module:
+  if (WiFi.status() == WL_NO_MODULE) {
+    Serial.println("Communication with WiFi module failed!");
+    // don't continue
+    while (true);
+  }
+
+  // Print firmware version on the module
+  String fv = WiFi.firmwareVersion();
+  String latestFv;
+  Serial.print("Firmware version installed: ");
+  Serial.println(fv);
+
+  latestFv = WIFI_FIRMWARE_LATEST_MODEL;
```

Comment:
This line should be `WIFI_FIRMWARE_LATEST_VERSION`.

Please also add a line to the Travis CI config to track build errors for the new sketch: https://github.com/arduino-libraries/WiFiNINA/blob/master/.travis.yml#L43